### PR TITLE
Update instructions for Always Blocked page

### PR DESCRIPTION
**Pull request by apmadz**
reference: https://github.com/Focus-Bear/web_dashboard/issues/821

### CODE REVIEW

File: focus-blocked/script.js [hunk 1]
```diff
@@ -72,14 +72,30 @@ const focus_tip = `<div class='notice-wrapper'>
...
   if (block_type === 'always-block' || block_type === 'always-blocked') {
-    document.getElementById(
-      'progressWrapper'
-    ).innerHTML = `<div class='notice-wrapper'><h6 class='centeredText'>${old_url} is configured to be always blocked. If you want to allow ${old_url}, go to Preferences > Always Blocked URLs</h6></div>`;
+    
+    /* Determine the operating system */
+    let userAgent = window.navigator.userAgent;
+    let os = "Unknown";                     
+    if (userAgent.indexOf("Win") != -1) os = "Windows";
+    if (userAgent.indexOf("Mac") != -1) os = "Mac";
```
Comment (amanueltsfy) [inline focus-blocked/script.js:82]:
```
if(userAgent.includes('Windows'){
  os = 'Windows'
}else if(userAgent.includes('Mac'){
  os = 'Mac'
}else {
  os = 'Unknown'
}
```

File: focus-blocked/script.js [hunk 2]
```diff
@@ -72,14 +72,30 @@ const focus_tip = `<div class='notice-wrapper'>
...
+    if (os == 'Mac' || os == 'Unknown') {
+      imgElement.src = 'https://images.focusbear.io/focusbear_blocksites.jpg';
+      document.getElementById('progressWrapper').innerHTML = `<div class='notice-wrapper'><h6 class='centeredText'>${old_url} is configured to be always blocked. If you want to allow ${old_url}, go to:<br>
+      Preferences > Settings > Strictness > Super Distracting Sites</h6></div>`;
+    } else if (os == 'Windows') {
+      imgElement.src = 'https://images.focusbear.io/focusbear_blocksites_windows.png';
+      document.getElementById('progressWrapper').innerHTML = `<div class='notice-wrapper'><h6 class='centeredText'>${old_url} is configured to be always blocked. If you want to allow ${old_url}, go to:
+      <br>Preferences > Settings > Blocked URLs > Super Distracting Sites</h6></div>`;
+    }
+
```
Comment (amanueltsfy) [inline focus-blocked/script.js:96]: if (os == 'Windows') {
      imgElement.src = 'https://images.focusbear.io/focusbear_blocksites_windows.png';
      document.getElementById('progressWrapper').innerHTML = `<div class='notice-wrapper'><h6 class='centeredText'>${old_url} is configured to be always blocked. If you want to allow ${old_url}, go to:
      <br>Preferences > Settings > Blocked URLs > Super Distracting Sites</h6></div>`;
    }
else {
      imgElement.src = 'https://images.focusbear.io/focusbear_blocksites.jpg';
      document.getElementById('progressWrapper').innerHTML = `<div class='notice-wrapper'><h6 class='centeredText'>${old_url} is configured to be always blocked. If you want to allow ${old_url}, go to:<br>
      Preferences > Settings > Strictness > Super Distracting Sites</h6></div>`;
}

**Comment by jeznag** [comment]
Could you share a video please?

**Comment by amanueltsfy** [comment]
> Could you share a video please?

https://www.loom.com/share/adbafba18ddf40ecb7d1de693f302988?sid=3b8911a0-3042-4893-8959-e1bd3f0fc7ff

**Pull request merged**